### PR TITLE
[Doc] PR for Document LDAP support for Platform ( CLI / API ) - Part 2

### PR DESCRIPTION
**Pull request by lizayugabyte**
Addresses https://yugabyte.atlassian.net/browse/PLAT-2730

Relates to https://github.com/yugabyte/yugabyte-db/pull/11023

Preview: https://deploy-preview-11111--infallible-bardeen-164bc9.netlify.app/latest/yugabyte-platform/administer-yugabyte-platform/ldap-authentication/

### CODE REVIEW

**Comment by lizayugabyte** [comment]
@sridhar-yb, kc-yb, Arnav15, could you, please, review the doc so I would be able to merge the PR in time for the release

**Comment by lizayugabyte** [comment]
@TaranSinghania-yb With 5 reviewers on the list, I don't think we'd be able to finish the doc in time for 2.12 release. If the file changes delivered by this PR are fine, I'm going to merge. Once the UI is available, I'll update the doc and create a new PR.

**Comment by TaranSinghania-yb** [comment]
Hi @lizayugabyte, we just need to wait for @Arpit-yb's review. We need to add some Windows AD specific information here.

**Review by Arpit-yb — CHANGES_REQUESTED** [review]
Can we add something for windows AD like setup, something like - "The user should have permissions to query its attributes/properties from the LDAP server otherwise the Platform will default its role as ReadOnly"

File: docs/content/latest/yugabyte-platform/administer-yugabyte-platform/ldap-authentication.md
Comment (lizayugabyte) [reply docs/content/latest/yugabyte-platform/administer-yugabyte-platform/ldap-authentication.md:104]: Deleted this sentence.

**Comment by lizayugabyte** [comment]
@TaranSinghania-yb @Arpit-yb Could you take a look at the latest changes and let me know if this PR is good to go

File: docs/content/latest/yugabyte-platform/administer-yugabyte-platform/ldap-authentication.md
Comment (Arpit-yb) [inline docs/content/latest/yugabyte-platform/administer-yugabyte-platform/ldap-authentication.md:108]: "the user is expected to query its properties"
Replace "AD's properties" -> "user's properties"

Rest everything looks good

**Review by Arpit-yb — COMMENTED** [review]
PR looks OK to me, @lizayugabyte just modify this minor line and we should be good to merge.
Will wait for @TaranSinghania-yb's approval

File: docs/content/latest/yugabyte-platform/administer-yugabyte-platform/ldap-authentication.md
Comment (lizayugabyte) [reply docs/content/latest/yugabyte-platform/administer-yugabyte-platform/ldap-authentication.md:108]: changed AD's to user's

**Comment by lizayugabyte** [comment]
@TaranSinghania-yb Could you approve the PR, please?

**Review by TaranSinghania-yb — CHANGES_REQUESTED** [review]

File: docs/content/latest/yugabyte-platform/administer-yugabyte-platform/ldap-authentication.md
Comment (lizayugabyte) [reply docs/content/latest/yugabyte-platform/administer-yugabyte-platform/ldap-authentication.md:60]: Added this example as:

Replace `[LDAP DN]` with the actual value, as per the following example: 
`,DC=yugabyte,DC=com`

File: docs/content/latest/yugabyte-platform/administer-yugabyte-platform/ldap-authentication.md
Comment (TaranSinghania-yb) [reply docs/content/latest/yugabyte-platform/administer-yugabyte-platform/ldap-authentication.md:82]: Basically:` LDAP DN Prefix + user name (entered in login form) + Base DN` - concatenation

Comment (lizayugabyte) [reply docs/content/latest/yugabyte-platform/administer-yugabyte-platform/ldap-authentication.md:82]: Added these two points

**Review by TaranSinghania-yb — APPROVED** [review]
LGTM for API only. Thanks so much!

**Review by Arpit-yb — APPROVED** [review]
LGTM, Nice work @lizayugabyte!

**Pull request merged**
